scripts/welding_schema.py

```diff
@@ -265,8 +263,12 @@ def single_pass_weld_example(out_file="single_pass_weld_example.asdf"):
         reference_timestamp=reference_timestamp,
         equipment=[HKS_sensor, BH_ELM],
         measurements=[welding_current, welding_voltage],
-        welding_current=current_calib_transform.output_signal,
-        welding_voltage=voltage_calib_transform.output_signal,
+        welding_current=welding_current_chain.get_signal(
+            "Calibration current measurement"
```

Comment:
could we also write something like get_signal(-1) to get the last entry of the chain? I think getting by name is nice, but seems a bit unhandy from a programmers point of view.

Reply:
would rather make it return the last signal of the chain if no name is provided like it is the case for some of the other functions. Since the underlying structure is not a list, a direct access operator doesn't make that much sense, does it? Never tried `dict[-1]`. Is that possible in python?

Reply:
I think it would be best and most intuitive to implement `.signals` and `.transformations` that expose a list of signals/tf or (maybe even better) provide an iterator?

That should enable `.signals[-1]`, `.signals[0:3]` etc

Reply:
Not sure, why an iterator would be better, but I have added the properties to return a list. We can change that to an iterator if necessary